node/clients/sellers.ts

```diff
@@ -19,7 +39,76 @@ export default class SellersClient extends JanusClient {
     })
   }
 
-  public async getSellers(): Promise<{ items: Seller[] }> {
-    return this.http.get(SELLERS_PATH)
+  public async getSellers(opts?: GetSellersOpts): Promise<GetSellersResponse> {
+    const argsWithDefaults = {
+      ...INITIAL_LIST_OPTIONS,
+      ...opts,
+    }
+
+    const result = await this.http.get<{
+      paging: {
+        from: number
+        to: number
+        total: number
+      }
+      items: Seller[]
+    }>(SELLERS_PATH, {
+      metric: 'sellers-get',
+      params: {
+        from: (argsWithDefaults.page - 1) * argsWithDefaults.pageSize,
```

Comment:
what if `page = 0`? what happens? do we need to protect against this case?
I also wonder what happens if we provide other invalid values like `from < to`.
Can you check how the sellers API behave when such cases happens? Maybe we can let the sellers API handle it, or we might need to check these values before sending it to the sellers API.

Reply:
Good point! The API returns a 400 bad request error in both cases, without any message explaining what caused the error. I'm going to return an exception in the case of `from` and `to` to make it clear to whoever is using it what caused the error.